### PR TITLE
Navbar bug fixed

### LINKED ISSUE #208: BUG: Navbar is not responsive on mobile version.
### Is there an existing issue for this?

- [X] I have searched the existing issues

## Description
- The navbar is not responsive in mobile version.
- The content stays as per the dimension, but the Navbar is overflowing.

## Preview
image

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a gssoc-24-extd contributor
- [X] I want to work on this issue
- [X] I check the issue doesn't exist

### PR DESCRIPTION
**Pull request by NilanchalaPanda**
## Description
- The navbar was previously overflowing with the whole content of the website.
- It was because of the Footer Content.

Fixes #208 

## Preview
image

## Summary by CodeRabbit

- **New Features**
	- Enhanced layout and responsiveness of the footer component.
	- Improved styling and structure of the Navbar and Landing components.
- **Bug Fixes**
	- Adjusted the height of the footer for improved visual consistency across screen sizes.
- **Refactor**
	- Simplified class structures for better readability and organization within the footer's navigation and content sections.
- **Documentation**
	- Updated README to clarify project goals, installation instructions, and contributor acknowledgments.

### CODE REVIEW

File: frontend/src/components/Shared/footer/Content.jsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React, { useEffect, useState } from 'react';
```
Comment (RamakrushnaBiswal) [inline frontend/src/components/Shared/footer/Content.jsx:1]: @NilanchalaPanda don't remove `React` other wise shows error while deployment

**Review by RamakrushnaBiswal — CHANGES_REQUESTED** [review]

**Comment by RamakrushnaBiswal** [comment]
@NilanchalaPanda also fix this 
image

**Comment by NilanchalaPanda** [comment]
Hey @RamakrushnaBiswal👋🏼

I have commented some part of the `index.css`, you can once read upon it to understand more about the issue. Other than that, I have resolved the bug majorly.

**Review by RamakrushnaBiswal — APPROVED** [review]

**Comment by RamakrushnaBiswal** [comment]
@NilanchalaPanda good work :)

**Comment by NilanchalaPanda** [comment]
Thanks @RamakrushnaBiswal, happy to contribute.

**Pull request merged**